content/pages/sprint/index.rst

```diff
@@ -13,12 +13,18 @@ hosted by sciwork.
 Scisprint 2024
 ==============
 
+* `(Tentative) Scisprint 2024 June in Taipei
+  <{filename}2024/06-taipei.rst>`__
+
 * `(Tentative) Scisprint 2024 May in Hsinchu
   <{filename}2024/05-hsinchu.rst>`__
 
 * `(Tentative) Scisprint 2024 April in Hsinchu
   <{filename}2024/04-hsinchu.rst>`__
 
+* `(Tentative) Scisprint 2024 March in Taipei
+  <{filename}2024/03-taipei.rst>`__
+
 * `(Tentative) Scisprint 2024 March in Hsinchu
```

Comment:
I remember that we cancel the March 23rd scisprint in Hsinchu. Please help us remove this page.